docs/source/api/v2/steering_id_targets_targetID.rst

```diff
@@ -1,230 +0,0 @@
-	Whole-Content-Sha512: AfXsIRzdtU3HZYkr93qBMVTZRJ5oTF2u5sKYnd+DSqxZ+RQxY6vXtCupnnXCf9dxMt5QXRW1EFOW/FBG6lFrTg==
-	X-Server-Name: traffic_ops_golang/
-	Date: Tue, 11 Dec 2018 14:34:22 GMT
-	Content-Length: 194
-
-	{ "alerts": [
-		{
-			"text": "steeringtarget was updated.",
-			"level": "success"
-		}
-	],
-	"response": {
-		"deliveryService": "test",
-		"deliveryServiceId": 2,
-		"target": "demo1",
-		"targetId": 1,
-		"type": "HTTP",
-		"typeId": 1,
-		"value": 1
-	}}
-
-``DELETE``
-==========
-Removes a specific target mapping from a specific :term:`Delivery Service`
-
-:Auth. Required: Yes
-:Roles Required: Portal, Steering, Federation, "operations" or "admin"
-:Response Type:  ``undefined``
-
-Request Structure
------------------
-.. table:: Request Path Parameters
-
-	+----------+----------------------------------------------------------------------------------------------------------------------------------------------------+
-	|   Name   |                Description                                                                                                                         |
-	+==========+====================================================================================================================================================+
-	|    ID    | The integral, unique identifier of a steering :term:`Delivery Service` - a target of which shall be deleted                                        |
-	+----------+----------------------------------------------------------------------------------------------------------------------------------------------------+
-	| targetID | The integral, unique identifier of a :term:`Delivery Service` which is a target to be removed of the :term:`Delivery Service` identified by ``ID`` |
-	+----------+----------------------------------------------------------------------------------------------------------------------------------------------------+
-
-.. code-block:: http
-	:caption: Request Example
-
-	DELETE /api/2.0/steering/2/targets/1 HTTP/1.1
-	Host: trafficops.infra.ciab.test
-	User-Agent: curl/7.47.0
-	Accept: */*
-	Cookie: mojolicious=...
-
-Response Structure
-------------------
-.. code-block:: http
-	:caption: Response Example
-
-	HTTP/1.1 200 OK
-	Access-Control-Allow-Credentials: true
-	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
-	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
-	Access-Control-Allow-Origin: *
-	Content-Type: application/json
-	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
-	Whole-Content-Sha512: N6h8Kl7uQveqpTc3fmKXFDY2yYe5smApNcaTow4ab0DHGFdJfqQh89I4nvvaXvmVNhxVAqX3UE/6blbO8/9Xqg==
-	X-Server-Name: traffic_ops_golang/
-	Date: Tue, 11 Dec 2018 14:42:54 GMT
-	Content-Length: 69
-
-	{ "alerts": [
-		{
-			"text": "steeringtarget was deleted.",
-			"level": "success"
-		}
-	]}
```

Comment:
You only deprecated the GET method, so all of this documentation should still be in there.